lbuild/facade.py

```diff
@@ -216,47 +274,120 @@ def ignore_files(*files):
     def ignore_paths(*paths):
         return lbuild.utils.ignore_patterns(*paths)
 
-    # deprecated
+
+    # deprecated functions
     @staticmethod
     def ignore_patterns(*patterns):
-        return lbuild.utils.ignore_patterns(*patterns)
+        deprecated("1.8.0", EnvironmentPostBuildFacade.ignore_patterns,
+                            EnvironmentPostBuildFacade.ignore_paths,
+                            obj=EnvironmentPostBuildFacade)
+        return EnvironmentPostBuildFacade.ignore_paths(*patterns)
 
-    # deprecated
     def get_generated_local_files(self, filterfunc=None):
+        deprecated("1.8.0", self.get_generated_local_files, self.generated_local_files)
         return self.generated_local_files(filterfunc)
 
-    # deprecated
     def outpath(self, *path, basepath=None):
-        return self._env.outpath(*path, basepath=basepath)
+        deprecated("1.8.0", self.outpath, self.real_outpath)
+        return self.real_outpath(os.path.join(*path), basepath=basepath)
+
+    def reloutpath(self, path, relative=None):
+        deprecated("1.8.0", self.reloutpath, self.relative_outpath)
+        return self.relative_outpath(path, relative)
+
+
+class EnvironmentPostBuildFacade(EnvironmentBuildCommonFacade):
+
+    def __init__(self, env):
+        EnvironmentBuildCommonFacade.__init__(self, env)
+
+    @property
+    def buildlog(self):
+        return self._env.facade_buildlog
+
+    def collector_values(self, key, unique=True):
+        collector = self._env.collectors.get(key)
+        if collector is not None:
+            return collector.values(unique)
+        return list()
+
+    def collector(self, key):
+        collector = self._env.collectors.get(key)
+        if collector is not None:
+            return CollectorFacade(collector)
+        return None
 
+    def collector_values_in_scope(self, key, filterfunc, unique=True):
```

Comment:
May `collector_values_filtered`? To me it is not clear what *in scope* means here.

Reply:
I can just add `filterfunc=None` to `collector_values()` and be done with it.

Reply:
That would also work.